tlatools/org.lamport.tlatools/src/tla2sany/parser/TLAplusParser.java

```diff
@@ -7768,6 +7770,22 @@ final private boolean jj_2_74(int xla) {
     finally { jj_save(73, xla); }
   }
 
+  final private boolean jj_3_52() {
```

Comment:
This change and the changes below are the result of non-determinism in the parser generator (JavaCC)?

Reply:
Correct.

Reply:
Could we eliminate this non-determinism? Reviewing PRs related to the parser would become much easier.

Reply:
Unfortunately probably not. The best way is to just ignore the auto-generated files in the PR diff. Unless github exposes functionality to not show diffs of certain files, the only other alternative is to remove the auto-generated files from source control entirely, which then has the issue that a clean checkout of the repository (before running the ant generate target) will show undefined symbol errors when opened in an IDE. However, I would support that if you want to do it.